### ISSUE TITLE
ModuleNotFoundError: No module named 'requests'

### ISSUE THREAD

**Issue by Sujithr1980** [Gioelio/Hack-Google-Password#1]
While trying to run python3 generator.py, I get the error as ModuleNotFoundError: No module named 'requests', I am trying to run this in Visual Studio Code. Can you please check.

Thanks & Regards,

Sujith

**Comment by Gioelio** [comment]
I think is because you don’t have python3 installed, but if you have
python3 installed and then you can’t run again i will try to resolve the
problem.

Thanks

**Comment by Sujithr1980** [comment]
I have Python version 3.8 installed.

I would also like to know when I try to enter $ python3 generator.py I get
a error, but after deleting "$" the script runs but get the error that I
previously mentioned in my email.

Thanks.

**Comment by Gioelio** [comment]
ok, maybe i forgot to write that you have to install the modules from pip, try run this command 
python3 -m pip install requests configparser
(anyway i want to remind you that you don’t have to copy $ when copy code, that character is only for indicate that the line is a line of cmd)

**Comment by Sujithr1980** [comment]
Thank you the script is working now :)

**Comment by Gioelio** [comment]
Thanks to you for your feedback, now i fix the readme, if it working good please give me a star

**Comment by Sujithr1980** [comment]
Sure...

**Comment by Sujithr1980** [comment]
Have an quick question

- Keywords about account (Split with ',')

Does this mean that if I have some keyword in mind I need to provide all
those keywords and split it with","
How about Special Characters like @ !, should this also be given with the
keywords
When it has the correct password will the script automatically stop?

**Comment by Gioelio** [comment]
Yes the idea is that you put some word and numbers(in numbers put also special character like @, !) and the script mix the word and send request, the script will save the attempted password in a file so that if you stop and then restart the script the old attempted password will not be reused, when the program find the password can be not stop, because it is multithreading and for light the thread this go alone, but when the password is found the script create another file with the password of the account

> Il giorno 25 mar 2020, alle ore 16:04, Sujithr1980 <notifications@github.com> ha scritto:
> 
> 
> Have an quick question
> 
> - Keywords about account (Split with ',')
> 
> Does this mean that if I have some keyword in mind I need to provide all
> those keywords and split it with","
> How about Special Characters like @ !, should this also be given with the
> keywords
> When it has the correct password will the script automatically stop?
>

**Comment by Sujithr1980** [comment]
Ok, I just checked the file which shows attempted password, I am able to
see the correct password of my account there, but as you said in your
previous email that script will create another file with the password of
the account. But this was not created and still the script is running.

**Comment by Gioelio** [comment]
I’ve tested this script so hard that when i publish it’s working, but i
thinks that now google had changed their control on the password, because
before it sends captcha to confirm that you’re not a robot (and i just sent
10 000 request without any one captcha) and i tryed to send the same
correct password of my account a lot of different times, but google respond
me that it was incorrect 499 times and 1 time told me with the token of the
account, now i will think about a solution to this problem and i let you
know when i discover something

Thanks a lot of much for your feedback

**Comment by Sujithr1980** [comment]
Thank you, i shall wait for your reply to restart the testing.
